src/test/java/teammates/test/cases/storage/FeedbackSessionsDbTest.java

```diff
@@ -211,6 +211,29 @@ private void testGetSoftDeletedFeedbackSessionsForCourse() {
         assertTrue(fsDb.getSoftDeletedFeedbackSessionsForCourse("idOfCourseNoEvals").isEmpty());
     }
 
+    @Test
+    public void testSoftDeleteFeedbackSession() throws Exception {
+        FeedbackSessionAttributes fs = getNewFeedbackSession();
+        fsDb.createEntity(fs);
+
+        ______TS("Success: soft delete an existing feedback session");
+        fsDb.softDeleteFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId());
+        FeedbackSessionAttributes deleted = fsDb.getFeedbackSession(fs.getCourseId(), fs.getFeedbackSessionName());
+
```

Comment:
`fsDb.getSoftDeletedFeedbackSession` should not return null.

Reply:
Do you mean I should use getSoftDeletedFeebackSession instead of getFeedbackSession for testing?

Reply:
Nope. Just add one more assertion.